blog.css

```diff
@@ -0,0 +1,342 @@
+body{
+  width: 100%;
+  height: 100%;
+  margin: 0px;
+  background-color: lightslategray;
+}
+h1{
+  width: 100%;
+  margin: 0pt;
+  margin-top: 200px;
+  margin-bottom: 50px;
+  text-align: center;
+  color: white;
+  font-size: 70px;
+}
+.headerbar{
+  display: flex;
+  justify-content: space-evenly;
+  background-color: white;
+  margin-top: 0px;
+  margin-bottom: 0px;
+  height: 50px;
+}
+.update{
+  background-color: black;
+  color: white;
+}
+.titlehead{
+  font-size: 40px;
+  background-color: white;
+  margin: 0px;
+}
+
+.toggle-button{
+  background-image: url(./imgs/Hamburger_icon.jpg);
+  width: 50px;
+  height: 50px;
+  border: none;
+  background-repeat: no-repeat ;
+}
+#sidebar {
+  position: fixed;
+  top: 0px;
+  width: 200px;
+  height: 100%;
+  background-color: lightslategray;
+  left: -200px;
+}
+#sidebar.active{
+  left: 0px;
+}
+#sidebar ul li {
+  color: white;
+  list-style: none;
+  padding: 15px 10px;
+  border-bottom: 1px solid black ;
+}
+#sidebar .toggle-button{
+  position: absolute;
+  left: 200px;
+  top:10px;
+}
+.toggle-button2{
+  background-color: snow;
+  width: 50px;
+  height: 50px;
+  border: none;
+  
+}
+#sidebar .toggle-button2{
+  display: flex;
+  
+}
+.search{
+  background: url(./imgs/search.png);
+  width: 50px;
+  height: 50px;
+  cursor: pointer;
+  font-weight: bold;
+  padding-bottom: 2px;
+  border: none;
+  background-color: none;
+}
+.nav-list{
+  padding-left: 0px;
+}
+
+.cart{
+  background: url(./imgs/shopping-cart-logo.jpg);
+  width: 85px;
+  height: 50px;
+  cursor: pointer;
+  font-weight: bold;
+  padding-bottom: 2px;
+  border: none;
+  background-color: none;
+  }
+.appointment{
+  border: line black;
+  border-width: 7px;
+}
+
+.maincontainer{
+  display: flex;
+  justify-content: center;
+}
+.mainbody{
+  width: 70%;
+  display: grid;
+  grid-template-rows: ;
+  grid-template-columns: 50% 50%;
+  grid-column-gap: 40px;
+  padding-bottom: 150px; grid-row-gap: 20px;
+}
+.blogcontainer{
+  margin-top: 20px;
+  margin-bottom: 20px;
```

Comment:
this might need a bit of padding. `padding: 20px;`
try it out and see if you like it!